www/src/pages/en/folder-structure.md

```diff
@@ -0,0 +1,281 @@
+
+<sub>(with NextAuth.js + tRPC)</sub>
+
+### `src/server/common/get-server-auth-session.ts`
+
+The `get-server-auth-session.ts` file is used to get the NextAuth.js session on the server-side. See [NextAuth.js usage](/en/usage/next-auth#usage-with-trpc) for more information.
+
+<sub>(with NextAuth.js + tRPC)</sub>
+
+## `src/server/db`
+
+The `db` folder contains the Prisma client.
+
+<sub>(with Prisma)</sub>
+
+### `src/server/db/client.ts`
+
+The `client.ts` file is used to instantiate the Prisma client at global scope. See [Prisma usage](/en/usage/prisma#prisma-client) for more information.
+
+<sub>(with Prisma)</sub>
+
+## `src/server/trpc`
+
+The `trpc` folder contains the tRPC server-side code.
+
+<sub>(with tRPC)</sub>
+
+### `src/server/trpc/context.ts`
+
+The `context.ts` file is used to create the context used in tRPC requests. See [tRPC usage](/en/usage/trpc#-servertrpccontextts) for more information.
+
+<sub>(with tRPC)</sub>
+
+### `src/server/trpc/trpc.ts`
+
+The `trpc.ts` file is used to export procedure helpers. See [tRPC usage](/en/usage/trpc#-servertrpctrpcts) for more information.
+
+<sub>(with tRPC)</sub>
+
+## `src/server/trpc/router`
+
+The `router` folder contains the tRPC routers.
+
+<sub>(with tRPC)</sub>
+
+### `src/server/trpc/router/_app.ts`
+
+The `_app.ts` file is used to merge tRPC routers and export them as a single router, as well as the type definitions. See [tRPC usage](/en/usage/trpc#-servertrpcrouterts) for more information.
+
+<sub>(with tRPC)</sub>
+
+### `src/server/trpc/router/auth.ts`
+
+The `auth.ts` file is an example tRPC router utilizing the `protectedProcedure` helper to demonstrate how to protect a tRPC route with NextAuth.js.
+
+<sub>(with NextAuth.js + tRPC)</sub>
+
+### `src/server/trpc/router/example.ts`
+
+The `example.ts` file is an example tRPC router utilizing the `publicProcedure` helper to demonstrate how to create a public tRPC route.
+
+<sub>(with tRPC)</sub>
+
+## `src/styles`
+
+The `styles` folder contains the global styles of the application, used by TailwindCSS.
+
+<sub>(with TailwindCSS)</sub>
+
+## `src/types`
+
+The `types` folder is used to store reused types or type declarations.
+
+<sub>(with NextAuth.js)</sub>
+
+### `src/types/next-auth.d.ts`
+
+The `next-auth.d.ts` file is used to extend the NextAuth default session type to include the user ID. See [NextAuth.js usage](/en/usage/next-auth#inclusion-of-userid-on-the-session) for more information.
+
+<sub>(with NextAuth.js)</sub>
+
+## `src/utils`
+
+The `utils` folder is used to store commonly re-used utility functions.
+
+<sub>(with tRPC)</sub>
+
+### `src/utils/trpc.ts`
+
+The `trpc.ts` file is the front-end entry point to tRPC. See [tRPC usage](/en/usage/trpc#-utilstrpcts) for more information.
+
+<sub>(with tRPC)</sub>
+
+## `.env`
+
+The `.env` file is used to store environment variables. See [Environment variables](/en/usage/environment-variables) for more information.
```

Comment:
```suggestion
The `.env` file is used to store environment variables. See [Environment variables](/usage/environment-variables) for more information.
```

Reply:
Fixed in https://github.com/t3-oss/create-t3-app/pull/731/commits/9a4ac76852359e187ea183f52165628b64e0d26d